frontend/src/store/modules/gems.ts

```diff
@@ -24,18 +24,18 @@ export interface GemsState {
   lastPickedUpGem?: Gem;
 }
 
-const initialDropGemFormState = Object.freeze({ color: GemColor.PURPLE });
-const initialState: GemsState = Object.freeze({
+const initialDropGemFormState = { color: GemColor.PURPLE };
+const initialState: GemsState = {
   gemsPendingCollection: [],
   dropGemFormState: initialDropGemFormState,
   lastPickedUpGem: undefined,
-});
+};
 
 const gemsModule: Module<GemsState, RootState> = {
-  state: initialState,
+  state: { ...initialState },
   mutations: {
     resetGemsStore(state) {
-      Object.assign(state, initialState);
+      Object.assign(state, JSON.parse(JSON.stringify(initialState)));
```

Comment:
What if we used [Lodash's `cloneDeep`](https://lodash.com/docs/#cloneDeep) instead? This isn't very obvious.

Reply:
was thinking it would be a bit overkil to add lodash just for this haha